src/pybind/mgr/dashboard/frontend/e2e/page-helper.po.ts

```diff
@@ -45,105 +93,142 @@ export abstract class PageHelper {
     return $('.datatable-footer-inner.selected-count');
   }
 
-  getTitleText() {
-    let title;
-    return browser
-      .wait(() => {
-        title = $('.panel-title');
-        return title.isPresent();
-      })
-      .then(() => title.getText());
-  }
-
-  getTableCell(content) {
+  // getTitleText() {
+  //   let title;
+  //   return browser
+  //     .wait(() => {
+  //       title = $('.panel-title');
+  //       return title.isPresent();
+  //     })
+  //     .then(() => title.getText());
+  // }
+  //
+  getTableCell(content: string): ElementFinder {
     return element(by.cssContainingText('.datatable-body-cell-label', content));
   }
 
-  getTable() {
-    return element.all(by.css('.datatable-body'));
+  getTable(): ElementFinder {
+    return $('.datatable-body');
   }
 
-  getTabsCount() {
+  async getTabsCount(): Promise<number> {
     return $$('.nav.nav-tabs li').count();
   }
 
-  getFirstTableCellWithText(content) {
+  getFirstTableCellWithText(content): ElementFinder {
     return element.all(by.cssContainingText('.datatable-body-cell-label', content)).first();
   }
 
   /**
-   * Used for instances where a modal container received the click rather than the desired element.
+   * Used for instances where a modal container would receive the click rather
```

Comment:
Is this line still related to this method?

Reply:
I don't think so, I'll remove it.